docs/src/components/ColorPalette.js

```diff
@@ -0,0 +1,43 @@
+// @flow strict
+import type { Node } from 'react';
+import { Box, Text } from 'gestalt';
+// $FlowFixMe[untyped-import]
+import tokens from 'gestalt-design-tokens/dist/json/variables.json';
+
+type Props = {|
+  name: string,
+  tokenId: string,
+|};
+
+const ColorPalette = ({ name, tokenId }: Props): Node => {
+  const tokenNumbers = [0, 50, 100, 200, 300, 400, 500, 600, 700, 800, 900];
+  const colorBoxes = tokenNumbers.map((number, idx) => {
+    const textColor = number > 400 ? 'white' : 'darkGray';
+    return tokens[`color-${tokenId}-${number}`] ? (
+      <Box
+        key={`${idx}-${tokenId}`}
```

Comment:
nit: this should probably use `${number}-${tokenId}`, since it's typically frowned upon to use indices as keys